Source/Dafny/Resolver.cs

```diff
@@ -12223,6 +12223,8 @@ private class RBranchExpr : RBranch {
         mti.UpdateBranchID(PB.Item2.BranchID, ctors.Count - 1);
       }
 
+      var ctorToFromBoundVar = new Dictionary<string, bool>();
```

Comment:
You can use a set here instead of a dictionary

Reply:
Good point.